azure_monitor/src/azure_monitor/utils.py

```diff
@@ -101,16 +111,17 @@ def __init__(
         ),
         storage_retention_period=7 * 24 * 60 * 60,
         timeout=10.0,  # networking timeout in seconds
+        **options,
```

Comment:
I believe we WANT to trigger an error if an unknown option is passed in. That is the whole point of having only key arguments in the constructor. We should supply a doc string for `Options` rather than accept any argument and not doing anything with the params.

Reply:
Well this was not the case before, so this would be a breaking change, why receive **options in the first place then, we need to update the code to receive an Options object

Reply:
Maybe receive either an Options object or the instrumentation key. If only receive the instrumentation, init Options with default values?

Reply:
We don't want this to be a breaking change, and creating an Options object and passing it into the exporter is adding another step into the instrumentation process (which is undesired). The only time where we "crash the app" in OpenCensus is if user instruments their exporters incorrectly. Passing in an unknown argument is not exactly "incorrect" but preventing them from doing so will probably save them and us a lot of time in the future, so having the exporter throw an error makes sense to me. 

If anything, we can catch this error and throw a nicer, more descriptive error message.

Thought?